testing/makefile

```diff
@@ -151,75 +151,53 @@ test_parser:
 	docker cp ${IMG_NAME}:/usr/local/testing/test_parser.log ./test_parser.log
 	make stop_jm
 
-test_data:
-# Compile testcase model
-	make compile_testcase_model TESTCASE=testcase2
-	make compile_testcase_model TESTCASE=testcase3
+test_data_generator:
 # Run jm docker container
 	make run_jm
 # Copy the required files and folders for the test
 	make copy_to_jm ARGS=data
-	make copy_to_jm ARGS=forecast
-	make copy_to_jm ARGS=kpis
 	make copy_testcase_to_jm TESTCASE=testcase2
-	make copy_testcase_to_jm TESTCASE=testcase3
-	docker cp ../testing/references/data/testcase2/kpis.json ${IMG_NAME}:/usr/local/testing/testcase2/models
-	docker cp ../testing/references/data/testcase3/kpis.json ${IMG_NAME}:/usr/local/testing/testcase3/models
 # Run test_data.py
```

Comment:
Change to `# Run test_data_generator.py`

Reply:
Fixed in https://github.com/ibpsa/project1-boptest/pull/229/commits/9e8978bbd65649d277ebb687fac302d94b584367.